src/includes/performance/default-sampling-context-platform/java.mdx

```diff
@@ -0,0 +1,16 @@
+For Java-based SDKs, it includes at least the following:
+
+```javascript
```

Comment:
JS?

Reply:
Just for the sake of formatting. But I agree that it could be confusing. I replace it with just links to javadocs.